sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/SamplesArguments.java

```diff
@@ -0,0 +1,69 @@
+package com.azure.digitaltwins.core;
+
+import org.apache.commons.cli.*;
+
+public class SamplesArguments {
+
+    private final String DIGITALTWINS_URL = "DigitalTwinsEndpoint";
+    private final String TENANT_ID = "tenantId";
+    private final String CLIENT_ID = "clientId";
+    private final String CLIENT_SECRET = "clientSecret";
+
+    private String digitalTwinUrl;
+    private String tenantId;
+    private String clientId;
+    private String clientSecret;
+
+    public SamplesArguments(String[] args) {
+        Options options = new Options();
+        Option input = new Option("d", DIGITALTWINS_URL, false, "DigitalTwins endpoint URL");
+        Option tenantId = new Option("t", TENANT_ID, false, "AAD Tenant Id");
+        Option clientId = new Option("c", CLIENT_ID, false, "AAD Client Id");
+        Option clientSecret = new Option("s", CLIENT_SECRET, false, "AAD Client Secret");
+
+        input.setRequired(true);
+        tenantId.setRequired(true);
+        clientId.setRequired(true);
+        clientSecret.setRequired(true);
+
+        options.addOption(input);
```

Comment:
nit: Can we do an inline addition to `options`? Just trying to reduce the lines of code here!

Reply:
sure, but won't save that much space :))

Reply:
```
        Options options = new Options()
            .addOption(input)
            .addOption(tenantId)
            .addOption(clientId)
            .addOption(clientSecret);
```

Reply:
How about:
```java
Options options = new Options()
            .addOption(new Option("d", DIGITALTWINS_URL, false, "DigitalTwins endpoint URL"))
            .addOption(new Option("t", TENANT_ID, false, "AAD Tenant Id"))
            .addOption(new Option("c", CLIENT_ID, false, "AAD Client Id"))
            .addOption(new Option("s", CLIENT_SECRET, false, "AAD Client Secret"));
```

There also seems to be a builder options:
```java
new Options()
	.addOption(Option.builder("d")
			.required()
			.hasArg(true)
			.argName(DIGITALTWINS_URL)
			.desc("DigitalTwins endpoint URL")
			.build())
	(...)
	.addOption(Option.builder("t")
			.required()
			.hasArg(true)
			.argName(TENANT_ID)
			.desc("AAD Tenant Id")
			.build());
```

However, I am not too particular about it, just giving you options :)